src/annotator/config/sidebar.js

```diff
@@ -1,6 +1,7 @@
 /**
  * Create the JSON-serializable subset of annotator configuration that should
  * be passed to the sidebar application.
+ * @param {Record<string, any>} config
```

Comment:
Can you leave a new line between the description and `@param` lines in JSDocs please.